setup.py

```diff
@@ -8,7 +8,7 @@
 setup(
     name='array_api_compat',
     version=array_api_compat.__version__,
-    packages=find_packages(include=['array_api_compat*']),
+    packages=find_namespace_packages(include=["array_api_compat*"]),
```

Comment:
What is the purpose of this change?

Reply:
``array_api_compat.dask`` doesn't contain an ``__init__.py`` file, so setuptools ended up skipping it when doing an install.

From the setuptools docs,
<img width="870" alt="image" src="https://github.com/data-apis/array-api-compat/assets/47963215/772f0d2c-03fc-4613-b24c-df085d989840">


IIUC, this doesn't show up in the CI since the CI runs array-api-compat from inside the checked out repo.

Reply:
Just add an `__init__.py` file. I don't think it's a good idea to mess with namespace packages.

Reply:
Gotcha, I reverted the change.